src/main/java/io/reactivesw/category/infrastructure/validator/CategoryNameValidator.java

```diff
@@ -58,8 +70,8 @@ public static void validateEqual(LocalizedString name, List<Category> sameRootCa
       String value = entry.getValue().toString();
       for (LocalizedStringValue categoryName : categoryNames) {
         if (key.equals(categoryName.getLanguage()) && value.equals(categoryName.getText())) {
-          LOG.debug("can not create category with same name : {}, key: {}", value, key);
-          throw new AlreadyExistException("Can not create category with same name");
+          LOG.debug("can not create category with same name : {}, key: {}.", value, key);
+          throw new AlreadyExistException("Can not create category with same name.");
         }
```

Comment:
1. Capitalize word in the beginning of log.
2. Remove space in front of punctuation